code/__HELPERS/names.dm

```diff
@@ -59,7 +59,7 @@ GLOBAL_VAR(command_name)
 		var/newname
 		if(config && config.station_name)
 			newname = config.station_name
-		else
+		else if(config)

```

Comment:
If this works this is a band-aid fix for a deeper problem, did you test this and were there any runtimes?